client/e2e/pages/EventsPage.ts

```diff
@@ -193,34 +227,58 @@ export class EventsPage {
     await this.editTicketQuantity.fill(showing.showingQuantity);
     await this.newEventSave.click();
     await this.eventContinue.click();
+   // await this.page.reload();
   }
 
   /**
    * We need to pass in things to the filter like:
    * "Wed, Oct 11 2023"
    */
-  async searchDeleteShowing(time: string) {
-    await this.showingCard
-      .filter({hasText: time})
-      .getByRole('button', {name: 'Edit'})
-      .click();
+  async searchDeleteShowing(Time: string) {
+   const showingCardLocator =  this.showingCard
+      .filter({hasText: Time})
+      .getByRole('button', {name: 'Edit'}); 
+   const disabled = await showingCardLocator.isDisabled();
+    //console out the value of the disabled//
+    console.log("disabled is: " + disabled); 
+    if (disabled)
+    {
+      await this.page.reload();
+    }
+    await showingCardLocator.click();
     await this.deleteShowingButton.click();
     await this.eventContinue.click();
   }
 
-  //**The weird thing is that Im not sure wheres the 'Playbill' comes from.
-  async checkNewEventOnHomePage() {
+  /**
+   * 1.Whenever we created a new event and go back to the homepage 
+   * since the homepage needs time to update, but the tests cant wait for too long
+   * We need to click the "cart" button in order to refresh the homepage then we can 
+   * see our newly created event on homepage now 
+   * 2. We need to wait for few seconds each time we click the slide
+   * since if we click it too fast, the image on homepage could only move 
+   * very little distance
+   */
+  async checkNewEventOnHomePage(event_name:string, suffix:string) {
     await this.homePage.click();
+    await this.page.waitForTimeout(100);
+    await this.cartButton.click();

```

Comment:
can you please check and make sure you need to go to cart? If anything, use a `this.page.reload()` instead. Is a reload needed? can you please check again?

Reply:
i dont think suffix is being used here?